pkg/sql/sqlstats/persistedsqlstats/flush_test.go

```diff
@@ -705,7 +704,7 @@ func TestSQLStatsReadLimitSizeOnLockedTable(t *testing.T) {
 
 	// Set table size check interval to .0000001 second. So the next check doesn't
 	// use the cached value.
-	sqlConn.Exec(t, "SET CLUSTER SETTING sql.stats.limit_table_size_check.interval='.0000001s'")
+	persistedsqlstats.SQLStatsLimitTableCheckInterval.Override(ctx, &s.ClusterSettings().SV, time.Nanosecond)
```

Comment:
Why did we need to override this instead of using the set statement above?

Reply:
The new code does the same thing. Override just sets the setting value directly.
This is an unnecessary change but one that simplifies test execution by removing the need for a full SQL parse/exec loop when we have the ability to set the value directly.